myblog/views.py

```diff
@@ -0,0 +1,42 @@
+from django.http import HttpResponse, JsonResponse
+
+from myblog.models import Post, PostView
+
+
+def getPosts(request):
+    posts_to_return = list();  # Make an empty list
+
+    m = 0
+    while True:
+        m = m + 1
+        if m >= Post.objects.count():
+            break;
+
+        l = Post.objects.all()[m];
```

Comment:
You should be able to send the Post collection to a serializer rather than looping through the collection and manually building the response. See: https://docs.djangoproject.com/en/1.10/topics/serialization/